pkg/api/bzz.go

```diff
@@ -147,10 +147,16 @@ func (s *Service) fileUploadHandler(w http.ResponseWriter, r *http.Request, stor
 		return
 	}
 
+	// filename cannot contain a "/" in prefix because the specification is that we cannot start with slash but can have a slash at a later position
+	if strings.HasPrefix(fileName, "/") {
+		logger.Debug("bzz upload file: / in prefix not allowed", "file_name", fileName, "error", err)
+		logger.Error(nil, "bzz upload file: / in prefix not allowed", "file_name", fileName)
```

Comment:
Is `logger.Error` necessary here? It looks like this is normal flow for handling requests which has early return for this case when filename starts with `/`. In other words error has not occurred in the system so `logger.Error` seams to be escalating severity.